subsys/net/ip/Kconfig.ipv4

```diff
@@ -34,7 +34,7 @@ config NET_IF_UNICAST_IPV4_ADDR_COUNT
 
 config NET_IF_MCAST_IPV4_ADDR_COUNT
 	int "Max number of multicast IPv4 addresses per network interface"
-	default 1
+	default 2
```

Comment:
I would do this a bit differently so that by default we would only have 1 multicast address struct (to save space) and only have 2 if IGMP is enabled.
```
default 2 if NET_IPV4_IGMP
default 1
```

Reply:
Yes indeed this makes sense, I've updated the code.